test/gtest/miopendriver_conv2d_trans.cpp

```diff
@@ -77,28 +81,48 @@ bool IsTestSupportedForDevice()
     return ::IsTestSupportedForDevMask<d_mask, e_mask>();
 }
 
-void RunMIOpenDriver()
+void RunMIOpenDriver(const std::string& floatArg, const std::vector<TestCase>& testCases)
 {
-    bool runTestSuite = miopen::IsEnabled(MIOPEN_ENV(MIOPEN_TEST_WITH_MIOPENDRIVER)) &&
-                        IsTestSupportedForDevice() &&
-                        miopen::IsEnabled(MIOPEN_ENV(MIOPEN_TEST_ALL)) &&
-                        (miopen::GetStringEnv(MIOPEN_ENV(MIOPEN_TEST_FLOAT_ARG)) == "--float" ||
-                         miopen::GetStringEnv(MIOPEN_ENV(MIOPEN_TEST_FLOAT_ARG)) == "--half" ||
-                         miopen::GetStringEnv(MIOPEN_ENV(MIOPEN_TEST_FLOAT_ARG)) == "--bfloat16");
+    bool runTestSuite = IsTestSupportedForDevice() &&
```

Comment:
I don't quite understand this logic (as I haven't worked with this code). I'd appreciate a descriptive comment that explains what `standalone` mode means etc.

Reply:
Updated comment to be more descriptive. 
Please take a look when you get a chance, and let me know if that helps clarify!

Reply:
ok, this makes sense. Thank you.